mllib/src/test/scala/org/apache/spark/mllib/tree/LossesSuite.scala

```diff
@@ -0,0 +1,22 @@
+package org.apache.spark.mllib.tree
```

Comment:
(Copyright header here). Looks reasonable but I don't know the reason for the difference in the first place. This `Losses` class doesn't seem to be used anyway? I can't find it at least.